decidim-core/db/migrate/20170307084957_create_reports.rb

```diff
@@ -0,0 +1,15 @@
+class CreateReports < ActiveRecord::Migration[5.0]
+  def change
+    create_table :decidim_reports do |t|
```

Comment:
There is a migration bellow to migrate existing `proposal_reports` to `reports`.